core/templates/components/profile-link-directives/circular-image.component.html

```diff
@@ -0,0 +1,5 @@
+<a *ngIf="isLinkAvailable()" [href]="link">
+  <img [src]="src" alt="" class="rounded-circle">
```

Comment:
I don't see an src variable in CircularImageComponent class in `circular-image.component.ts`.